### DISCUSSION TITLE
Only electron, no ion in Species block

### DISCUSSION THREAD

**Discussion by SillyTom** [SmileiPIC/Smilei#735] [Q&A] [answered]
For many example laser wake code files, like 'tst3d_v_o2_laser_wake_yee_boris,py', only electron is defined in the 'Species' block, why is there no ion in plasma?

**Comment by mccoys** [comment]
@SillyTom Please do not use issues for questions. Use the chatroom or these discussions.

**Comment by Z10Frank** [comment, marked as answer]
@SillyTom You can find the answer in the section `A subtlety: why ions are not present` in this tutorial: https://smileipic.github.io/tutorials/advanced_wakefield_AMcylindrical.html#a-subtlety-why-ions-are-not-present
